### ISSUE TITLE
How to use translation for Custom Options

### ISSUE THREAD

**Issue by devzorg** [dweeves/magmi-git#509]
Information is not enough on the page http://wiki.magmi.org/index.php?title=Custom_Options

How to translate Can you post the demo how to translate Title of option and Values of options and how to link each other?

**Comment by devzorg** [comment]
e.g. that doesn't work
header `store,Size:drop_down:0:1`
items `
admin,XS:fixed:0:XS:1
ru,[Размер]:Extra Small:fixed:0:XS:1

How to translate XS? in RU store_id I see unchecked Default option and the same value XS only and Option Title also the same default value but unchecked Default
`

**Comment by YorrickDL** [comment]
I got it working like this:

```
store,sku,fs_wine_wijnzaak_card_label:radio:0:1
admin,WZ50015,Greeting card:fixed:1.5:WZ50038:1|Greeting card attached to the bottle with a metal chain:fixed:1.99:WZ50039:2
wines_nl_nl,WZ50015,[fs_wine_wijnzaak_card_label]:Kaartje:fixed:1.5:WZ50038:1|Kaartje met ketting om hals fles:fixed:1.99:WZ50039:2
```

You have to make sure that your file is ordered by sku.

**Comment by devzorg** [comment]
@YorrickDL ok, you have translated values, but what about option name?
